public/app/features/explore/Explore.tsx

```diff
@@ -347,7 +347,16 @@ export class Explore extends React.PureComponent<ExploreProps, ExploreState> {
                 }
 
                 return (
-                  <main className={cx('m-t-2', styles.logsMain, styles.fullHeight)} style={{ width }}>
+                  <main
+                    className={cx(
+                      'm-t-2',
```

Comment:
not big fan of this utility class, can't it be part of the class definition?

Reply:
Yup can change that.